docs/resources/schedule_v2.md

```diff
@@ -64,7 +64,7 @@ resource "squadcast_schedule_v2" "schedule_test" {
 Required:
 
 - `id` (String) Schedule owner id.
-- `type` (String) Schedule owner type (user, team, squad).
```

Comment:
Use description like "supported values are "user", "team" and "squad"
This applies to all the files